velox/dwio/dwrf/test/TestColumnStatistics.cpp

```diff
@@ -602,50 +602,50 @@ TEST(StatisticsBuilder, basicMissingStats) {
 }
 
 TEST(StatisticsBuilder, basicHasNull) {
-  enum State { TRUE = 0, FALSE, MISSING };
+  enum State { TRUE_STATE = 0, FALSE_STATE, MISSING_STATE };
```

Comment:
@yingsu00, could you also use `enum class` here instead of a regular enum? Not sure what was the compilation issue you found, but this in itself might fix the issue. Thanks!

Reply:
@pedroerp using enum class doesn't help:
![Screen Shot 2021-12-31 at 1 28 20 AM](https://user-images.githubusercontent.com/33299678/147815678-6213a991-6330-47ef-8071-c43e702b2091.png)
![Screen Shot 2021-12-31 at 1 28 09 AM](https://user-images.githubusercontent.com/33299678/147815682-51827b08-79b1-48cf-bcd4-4e6f8da41cff.png)